docs/topics/templates/document-attributes.adoc

```diff
@@ -115,6 +115,8 @@
 :red-hat-ga-repo-name: Red Hat GA Repository
 :red-hat-ga-repo-url: https://maven.repository.redhat.com/ga/
 
+:link-spring-boot-guide: https://appdev.openshift.io/docs/spring-boot-runtime.html
```

Comment:
That's an old and obsolete URL, right? Should point to launcher.fabric8.io, right?